index.php

```diff
@@ -0,0 +1,152 @@
+<?php
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+
+require __DIR__ . '/vendor/autoload.php';
+
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use JoliCode\Slack\ClientFactory;
+use JoliCode\Slack\Exception\SlackErrorResponse;
+require "agenda.php";
+require "security.php";
+
+$log = new Logger('SlackApp');
+$log->pushHandler(new StreamHandler('access.log', Logger::DEBUG));
+
+//
+// Checking credentials
+// 
+if(!file_exists('credentials.json')) {
+    $log->error('credentials.json not found');    
```

Comment:
shouldn't we also exit() here (as you do on all other invalid conditions below)?

Reply:
Indeed, fixed in f4d61db